packages/amplify-graphql-api-construct/.jsii

```diff
@@ -0,0 +1,5770 @@
+{
+  "author": {
+    "name": "Amazon Web Services",
+    "roles": [
+      "author"
+    ]
+  },
+  "bundled": {
+    "@aws-amplify/graphql-transformer": "1.1.2",
+    "@aws-amplify/graphql-transformer-core": "2.1.2",
+    "@aws-amplify/graphql-transformer-interfaces": "3.1.2",
```

Comment:
How do we ensure these get bumped in every release? When do we intend to refresh this file?

Reply:
This file is generated by the `build` step, so it'll be up-to-date.